spec.bs

```diff
@@ -4038,6 +4076,9 @@ The <dfn for="interest group">estimated size</dfn> of an [=interest group=] |ig|
     {{long}} integer, so 4 bytes.
 1. [=list/For each=] |key| of |ig|'s [=interest group/trusted bidding signals keys=]:
   1. The [=string/length=] of |key|.
+1. 4, which is the size of |ig|'s [=interest group/max trusted bidding signals url length=],

```

Comment:
nit: may keep it consistent with other fields:
If .. is not null: 4, which is ....